### DISCUSSION TITLE
How Logto different from Next-auth?

### DISCUSSION THREAD

**Discussion by kawpii** [logto-io/logto#2347] [Q&A] [answered]
As I understood Logto is full stack solution and Next Auth is similar to Logto client SDK, but you can make Next-auth a fullstack solution using custom DB, So my question is what is the benefit of Logto over Next-auth, than some time saving due to built in back-end and dashboard?

**Comment by FlurryNight** [comment, marked as answer]
🫣

Next Auth (as the name says) is a authentication solution for NextJS apps

Logto is an IAM solution that is easy to extend and use across different platforms.

**Reply by kawpii** [reply]
I think it is possible to use in Gatsby too.

https://github.com/nextauthjs/next-auth-gatsby-example

**Reply by FlurryNight** [reply]
yeah but they are different solutions
